autoPyTorch/pipeline/tabular_classification.py

```diff
@@ -196,9 +196,8 @@ def _get_hyperparameter_search_space(self,
             embeddings = cs.get_hyperparameter('network_embedding:__choice__').choices
             if 'LearnedEntityEmbedding' in embeddings:
                 encoders = cs.get_hyperparameter('encoder:__choice__').choices
-                default = cs.get_hyperparameter('network_embedding:__choice__').default_value
                 possible_default_embeddings = copy.copy(list(embeddings))
-                del possible_default_embeddings[possible_default_embeddings.index(default)]
+                del possible_default_embeddings[possible_default_embeddings.index('LearnedEntityEmbedding')]
```

Comment:
Can you change the comment before that says:
`Here we add custom code, like this with this`